### PR TITLE
fix type error in llm collection

### PR DESCRIPTION
**Pull request by stevehuang52**
# What does this PR do ?

fix type error in llm collection

### CODE REVIEW

File: nemo/lightning/io/mixin.py [hunk 1]
```diff
@@ -326,6 +326,7 @@ def _get_connector(cls, ext, path=None, importer=True, **kwargs) -> ModelConnect
                         when required.
         """
         _path = None
+        ext = str(ext)
```
Comment (maanug-nv) [inline nemo/lightning/io/mixin.py:329]: Docstring already suggests this should be a string, but no harm in this I guess. Can we add type hints for the args as well?

Comment (stevehuang52) [reply nemo/lightning/io/mixin.py:329]: Added

Comment (maanug-nv) [reply nemo/lightning/io/mixin.py:329]: sorry for nitpick, can you update docstring type of `path` to match signature typehint (or vice versa), for consistency.

Comment (stevehuang52) [reply nemo/lightning/io/mixin.py:329]: Done~

**Review by maanug-nv — APPROVED** [review]
All concerns resolved

**Pull request merged**